### ISSUE TITLE
Arcane Tracker opening itself and running in the background

### ISSUE THREAD

**Issue by Beebles00** [HearthSim/arcane_tracker#92]
The app keeps popping up and running in the background, despite closing it each time it pops up and without having hearthstone even running. A notification keeps showing in the upper notification bar on the latest Android version (on a Galaxy S8) where I am able to quit. Issue is happening seemingly randomly, I haven't been able to pinpoint any kind of trigger for when this happens.

**Comment by NewtonHorse** [comment]
I am having the same issue with a Galaxy S10. I was able to get Arcane Tracker to close by switching my phone to Landscape mode and tapping "Quit" in the Arcane Tracker settings (while not in Hearthstone). Hopefully this helps.

**Comment by Xuton** [comment]
yup, got the same issue on s9's and s20's. Haven't opened heartstone in days, yet the app keeps popping up in my system tray.

I see that this issue was opened ages ago. Anyone looking into it?
